torch_ort_inference/docker/Dockerfile.ort-infer-stable-torch1120-onnxruntime1120-openvino-ubuntu18.04

```diff
@@ -0,0 +1,85 @@
+#-------------------------------------------------------------------------
+# Copyright(C) 2022 Intel Corporation.
+# SPDX-License-Identifier: MIT
+#--------------------------------------------------------------------------
+
+ARG OPENVINO_VERSION="2022.1.0"
+ARG OVPT_BRANCH="releases/v0.5"
+
+FROM openvino/ubuntu18_runtime:${OPENVINO_VERSION}
+
+LABEL description="This is the runtime image for OpenVINO™ integration with PyTorch on Ubuntu 18.04 LTS"
+
+LABEL vendor="Intel Corporation"
+
+USER root
+
```

Comment:
RUN dpkg --get-selections | grep -v deinstall | awk '{print $1}' > base_packages.txt missing.

Reply:
fixed, please review for any discrepancies.